api/lambda_function.py

```diff
@@ -11,8 +11,17 @@ def lambda_handler(event, context):
     request_type = event['requestType']
     store_id = event['storeId']
     customer_id = event['customerId']
-
-    response = {}
+    # write store_id & customer_id to the DynamoDB table using the object we instantiated and save response in a variable
+    response = table.put_item(
+        Item={
+            'storeId': store_id
+            'customerId':customer_id
+            })
+    # return a properly formatted JSON object
+    return {
```

Comment:
Having a return here will mean that nothing below this will ever execute which is not what we want